packages/ui/src/components/file-explorer.tsx

```diff
@@ -9,27 +9,16 @@ interface FolderItemProps {
   value?: string
   isActive?: boolean
   content?: ReactNode
-  chevronClassName?: string
   link: string
 }
 
-function FolderItem({ children, value = '', isActive, content, chevronClassName, link }: FolderItemProps) {
+function FolderItem({ children, value = '', isActive, content, link }: FolderItemProps) {
   const { Link } = useRouterContext()
   return (
     <Accordion.Item value={value} className="border-none">
       <Accordion.Trigger
-        className={cn(
-          `relative w-full p-0 pr-1.5
-          before:absolute before:z-[-1] before:top-0 before:-left-1.5 before:right-0 before:h-full before:rounded`,
-          {
-            'before:bg-cn-background-hover': isActive
-          }
-        )}
-        leftChevron
-        rotateChevron
-        chevronClassName={
-          chevronClassName || 'text-icons-9 group-hover:text-icons-2 group-data-[state=open]:text-icons-2'
-        }
+        className="relative w-full p-0 pr-1.5 [&>.cn-accordion-trigger-indicator]:mt-0 [&>.cn-accordion-trigger-indicator]:-rotate-90 [&>.cn-accordion-trigger-indicator]:self-center [&>.cn-accordion-trigger-indicator]:data-[state=open]:-rotate-0"
+        indicatorProps={{ size: 10 }}
```

Comment:
Would this instance be badly effected with the standard 14 size?

Reply:
btw, the icon size here should be 12